Makefile

```diff
@@ -122,25 +122,21 @@ TOOLS := "$(TOOLS) uv"
 uv-tooling:
 	pip install --user uv
 uv-import:
-	cat requirements.txt
+	cd uv; uv add --frozen -r ../requirements.txt
 uv-clean-cache:
 	rm -rf ~/.cache/uv
 uv-clean-venv:
 	rm -rf uv/.venv
 uv-clean-lock:
-	rm -f uv/requirements.txt
+	rm -f uv/uv.lock
 uv-lock:
-	mkdir -p uv
-	uv pip compile --generate-hashes --output-file=uv/requirements.txt requirements.txt
+	cd uv; uv lock
 uv-install:
-	test -f uv/.venv/bin/python || uv venv uv/.venv
-	VIRTUAL_ENV=$$(pwd)/uv/.venv uv pip sync uv/requirements.txt
+	cd uv; uv sync
 uv-update:
-	uv pip compile --output-file=uv/requirements.txt requirements.txt
-	VIRTUAL_ENV=$$(pwd)/uv/.venv uv pip sync uv/requirements.txt
+	cd uv; uv lock --upgrade
```

Comment:
Should be `uv sync --upgrade` for equivalence with other tools

Reply:
Are you pointing out against `uv lock --upgrade` or against `uv sync`?
(If the former, it seems to be the wrong part of the review.)

Can you tell me specifically which commands you are comparing to those of other tools?

Reply:
All of the tool-update commands refresh both the lock file and the virtual environment.  So you want `uv sync`,  not only `uv lock`

Reply:
It is true that the `update` benchmark requires you to install the package in addition to resolving dependencies.
I will fix this later.